book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+    * Consider the best places to advertise to reach a diverse pool of candidates
+        * See “ Resources” section for suggestions of advertising venues
+        * If certain demographics are often missing from your pool, utilise spaces that target those groups
+        * Consider reviewing the diversity of the pool attained from previous, similar advertising and seek specific advertising to reach other groups that were underrepresented 
+    * Use a gender decoder to review the advert after you have created it
+    * Leverage your networks, as well as those of others in the organisation, to share the opportunity advertised e.g. Slack, LinkedIn, word of mouth at conferences etc. 
+        * When utilising your networks, consider if there are any groups potentially missing. Consider using alternative spaces (examples listed in Resources section) to reach groups that aren’t in your network. 
+- [ ] Set a suitable closing deadline
+    * Ensure a closing date is included on the advert
+        * Advertise roles minimally for 2-4 weeks where possible
+    * Where possible, stick to the advertised close date. Avoid stating that the role will be closed early if a suitable candidate is identified.  
+    * Consider the day of the week and time of closing your advert. For example, a closing date set on a weekend may be valuable for people who have limited free time.
+    * Be explicit on the time zone which will be used to close the application period 
+    * 
+
+
+### Before job advert closes
+
+
+
+- [ ] Book time(s) for interview panel *[Contributing to this action may be out of scope for hiring managers depending on organisational processes]*
+    * Consider flexibility and other requirements for applicants AND members of the interview panel. For example, ensuring that panel members have sufficient  breaks and that flexibility in scheduling can be offered to applicants - planning should be undertaken in partnership with your HR team.  
+    * Ensure there is a diverse interview panel for every interview - consider gender, race, diversity of thought, interest/impacted group perspective. 
+    * Consider having the option of both in-person or remote interviews for candidates to select from
+        - [ ] Consider providing reimbursement for expenses incurred for candidates attending in-person interviews 
+- [ ] Decide on assessments and criteria
+    * Be clear on what criteria is being used for shortlisting
+    * Ensure assessment criteria focus on the person specification. This should help mitigate bias during hiring decisions later on.
+
+
+### At the deadline 
+
+
+
+- [ ] Close job advert 
+    * Be transparent with the status of the application and timelines
+    * Ensure reasonable timeframes are kept when making decisions 
+    * Either at this stage, or at the end of the process, review anonymised demographic data of candidates to see if your advertising avenues were ‘effective’, how ever this may be determined by your organisation
+
+
```

Comment:
Maybe add a section to say that, given the current situation on the job market and in general in education and society, it might (and probably will) happen that you try really hard and do all the right things, and still end up with a disappointingly non-diverse pool of applicants. This should not be taken as failure, and not deter you from trying hard again next time.

Reply:
You are being generous :). I risk moving beyond the "lightly opinionated" flavour of TTW if I say much more on this ;). Interested to know what other think. Khanisa in our EDI team is good on this (also just noticed that she's not on the author list!!) - I'll ask her to comment :).

Reply:
Added a sentence in the opening :). See [commit](https://github.com/the-turing-way/the-turing-way/pull/4070/commits/3672f6b999888dd450ad012ff7085ce79c34719b)

Reply:
@MarionBWeinzierl Please could you confirm that you are happy with the new text on line 14? We've steered away from saying *why* you might fail, and focused only on that you should not give up :).